src/txmempool.h

```diff
@@ -516,6 +516,19 @@ class CTxMemPool
     void ApplyDelta(const uint256& hash, CAmount &nFeeDelta) const EXCLUSIVE_LOCKS_REQUIRED(cs);
     void ClearPrioritisation(const uint256& hash) EXCLUSIVE_LOCKS_REQUIRED(cs);
 
+    struct delta_info {
+        /** Whether this transaction is in the mempool. */
+        const bool in_mempool;
```

Comment:
Seems strange to make these `const`. `const auto deltas = mempool.GetPrioritisedTransactions()` should be enough...

Reply:
Seems like it would be better to explicitly specify defaults (`false` and `0`?) in that case? If you want to delete the default constructor to force aggregate initialization, then `delta_info() = delete;` seems more direct?